File: benchmark/results/ts-monolith-sonnet-script/run-001/a-mcp/output/src/orders/fulfillment/services/carrier-gateway.ts
import { Shipment } from '../aggregates/shipment';
import { Id } from '../../../shared/value-objects/id';

export class CarrierGateway {

  create(input: Partial<Shipment>): Shipment { throw new Error('not implemented'); }

  findById(id: Id): Shipment | null { throw new Error('not implemented'); }

  list(limit: number): Shipment[] { throw new Error('not implemented'); }

  delete(id: Id): void { throw new Error('not implemented'); }
}

File: benchmark/results/python-multilang/run-002/a-mcp/output/shipping/services/carrier_gateway.py
from typing import Any, Dict, List, Optional
from shipping.aggregates.shipment import Shipment

"""CarrierGateway service."""
class CarrierGateway:
    def create(self, input: Dict[str, Any]) -> Shipment:
        raise NotImplementedError('not implemented')
    def findById(self, id: str) -> Optional[Shipment]:
        raise NotImplementedError('not implemented')
    def list(self, limit: int) -> List[Shipment]:
        raise NotImplementedError('not implemented')
    def delete(self, id: str) -> None:
        raise NotImplementedError('not implemented')

